go/libraries/doltcore/schema/alterschema/addcolumn.go

```diff
@@ -105,7 +104,7 @@ func updateTableWithNewSchema(ctx context.Context, tblName string, tbl *doltdb.T
 	}
 
 	err = rowData.Iter(ctx, func(k, v types.Value) (stop bool, err error) {
-		oldRow, _, err := tbl.GetRow(ctx, k.(types.Tuple), newSchema)
+		oldRow, err := row.FromNoms(newSchema, k.(types.Tuple), v.(types.Tuple))
```

Comment:
This is temporary for this change, we'll need a general purpose way to set defaults